docs/tutorials/uky_flowsheet-solution.ipynb

```diff
@@ -1660,6 +1655,14 @@
     "        m.fs.rougher_mixer,\n",
     "    ]\n",
     "\n",
+    "    initializer_sx = SolventExtractionInitializer()\n",
+    "    sx_units = [\n",
+    "        m.fs.solex_rougher_load,\n",
+    "        m.fs.solex_rougher_scrub,\n",
+    "        m.fs.solex_cleaner_load,\n",
+    "        m.fs.solex_cleaner_strip,\n",
+    "    ]\n",
+    "\n",
     "    # The BT Initializer will be used for any units not handled by the above initializers\n",
```

Comment:
As per the UKy flowsheet diagram, there is a rougher stripping unit also in the flowsheet, why is the rougher stripping unit not being initialized here?